api/messages.proto

```diff
@@ -97,6 +97,23 @@ message BoolEqualsFilter {
   bool value = 2;
 }
 
+// Filters strings exactly equaling a value.
+//   attribute: "foo"
```

Comment:
Can I specify value of my attribute to be null? Should we fail setting it?

Reply:
Added null to does not match list.